client/src/app/users/user-card.component.ts

```diff
@@ -6,14 +6,10 @@ import { User } from './user';
   templateUrl: './user-card.component.html',
   styleUrls: ['./user-card.component.scss']
 })
-export class UserCardComponent implements OnInit {
```

Comment:
We didn't actually do anything with `ngOnInit()` (it was empty, and `ng lint` didn't like that), so I removed the reference to it.